### PR TITLE
New post: interesting work, not interesting jobs

### PR DESCRIPTION
**Pull request by nickflux**
This has already been through an editing mill, so just looking for copy editing and release date approval (5/27/15).

### CODE REVIEW

**Comment by davetron5000** [comment]
@nickflux  looks good.  I made a few changes to the typography, notably formatting the list of interesting problems as a bullet list  Let me know if you didn't intend that.

File: _posts/2015-05-27-interesting-work-not-interesting-jobs.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+There are two general problems with interesting jobs. The first is that what seems interesting from the outside can turn out to be mostly tedious once you're doing it. I was a singer in many bands during my twenties. Playing live is incredible, and writing songs is pretty good (and presumably much better if you are talented), but most of the time you are rehearsing (playing the same song over and over and over again in a foul-smelling room, mostly being ignored by the actual musicians) or recording (mostly sitting on a foul-smelling sofa drinking tea). From what I can tell acting, especially in movies or on TV, has a similarly high sofa quotient.
+
+The second problem is that the perceived interestingness of a particular job is probably somewhat proportional to the number of people who want that job. If lots of people want to do the same job then competition will be intense which will inevitably lead to downward pressure on wages. Once again, see acting where unemployment rates are above 80%. Elsewhere in the film industry, [entry level jobs][runner-job-description], even for post-graduates, can pay minimum wage.
```
Comment (ndp) [inline _posts/2015-05-27-interesting-work-not-interesting-jobs.md]: a few commas before independent clauses will help. "will be intense *[comma]* which..."  and "see acting *[comma]* where"

**Comment by nickflux** [comment]
@davetron5000 thanks for catching it - I did intend that.

**Comment by nickflux** [comment]
@ndp changed for clarity

File: _posts/2015-05-27-interesting-work-not-interesting-jobs.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+If you are a software engineer you are unlikely to be looking at minimum wage for your first job—despite the media frenzy this industry is still not as interesting as the movies—but the competition for 'interesting' engineering jobs is likely to be strong. It's common for talented engineers to focus their job hunts on the hottest startups or highest profile tech companies. But they might have more luck doing a bit of ‘Moneyball’-style analysis to find companies that are undervalued by other software engineers.
+
+In this context _value_ means companies where your work will have a significant impact on the business. Software has a huge impact at Google, for example, but if you're the 20,000th sotware engineer how much marginal impact are you going to have? If, however, you are the 20th engineer at a mature business with money in its pockets and a leadership interested in innovation, it's likely that you can play a large part in whatever technology they decide to develop. You'll probably get a chance to help make those decisions.
```
Comment (emmafern) [inline _posts/2015-05-27-interesting-work-not-interesting-jobs.md]: typo on `20,000th sotware engineer`

File: _posts/2015-05-27-interesting-work-not-interesting-jobs.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+Of course, not many software professionals are desperate to work in the retail industry, but consider the problems we are trying to solve (to name just a few):
+
+* Stitch Fix has built a unique styling platform used by over 1,500 stylists who work 10 to 30 hrs/week. This platform, designed and built by the engineering team, allows stylists to work with a short list of styles, recommended by an algorithm specifically for each client, to pick five items the customer will love. How would you build a tool that can handle a different data set for every client as well as presenting all a client's previous shopping history, in a way that improves a stylist's efficiency and success rate?
+* Our customers give us enormous amount of data about themselves.  We also have a lot of data about  the clothes we sell. Our world-class [data science team][data-sci-team] produces insightful analysis about the interactions between the client and clothes. How can you  integrate real-time sales data and client feedback analysis into the tools where the buying teams are planning and placing orders for inventory six months from now?
```
Comment (emmafern) [inline _posts/2015-05-27-interesting-work-not-interesting-jobs.md]: amount should be amounts, or 'give us an enormous amount'

there's an extra space after 'data about __ the clothes', and 'How can you __ integrate'

'produces insightful analysis' could have 'analyses' instead

**Pull request merged**